src/main/resources/static/api.yml

```diff
@@ -2099,6 +2099,39 @@ paths:
           $ref: '#/components/responses/403Response'
         500:
           $ref: '#/components/responses/500Response'
+  /placement-requests/{id}/booking:
```

Comment:
Thinking about this - I wonder if it makes more sense to relate the booking to the Application, as this is the thing we _really_ care about. The PlacementRequest is more a record to aid with matching

Reply:
On reflection (and chatting to @brownbearsoftware) I realised that Applications can have multiple PlacementRequests, so linking through the request is the thing to do here